# Bi622: Week 3

## 08/04/2021  

1) Applications using HMW DNA  
* 10X Genomics - WG linked reads  

#

![](Figures/10X_genomics.png)  

#

* Pac-Bio  
    * Circular Consensus Sequencing (same as HiFi)
    * Each pass is called a subread
    * The full sequence of all subreads and adapters is called a polymerase read
    * Multiple passes allows for higher quality (&#8805; Q30 ~ 99.99% accuracy)  
#

![](Figures/circ_con.png)  

#
2) Why is undamaged DNA important?  
* Repetitive content requires large reads  
* Longest insert possible (&#62; 30-40-50+ kb)  
* Sequencing stats are reported for the longest subread  

3) HiFi
* Better genome assembly  
* Structual variant analysis  
    * Depends on the size of the variant
* Base modification analysis
* Small variant analysis (SNPs, indels)
* Amplicon sequencing
* Any other small input libraries (plasmids, target capture, etc.)
* Iso-Seq - full length transcript isoforms  
4) CLR  
* Genome assembly
* Structual variant analysis  
* Base modification analysis  
5) HMW DNA is easily damaged   
* Sensitive to mechanical force  
* Abasic sites, nicks and gaps, thymine dimers, blocked 3' ends, oxidized guanines/pyrimidines  
* Damaged DNA will prevent polymerase activity  
6) How to minimize damaage  
* No exposure to dyes, UV, high temp, extreme pHs or vortexing.
* Minimal time spent stored at RT or 4&#176; C (store at -20&#176;)  
* Minimize freeze/thaw cycles  
* Must mix thoroughly but gently
* Use Eppendorf Protein Lo-Bind tubes
* Take your time and be patient  

7) Considerations for lab:  
* Quality
* Quantity (+10 micrograms for shearing , +5 micrograms after)
* Purity - using nanodrop  
    * Use the refractive index of each type of molecule  to identify impurities  

8) DNA damage can be corrected to some degree during the preperation steps  

9) Circulomics Nanobind Extraction









